include/ppx/knob.h

```diff
@@ -400,6 +400,8 @@ class KnobFlag final
 
     T GetValue() const { return mValue; }
 
+    bool IsDefaultValue() const { return mDirty; }
```

Comment:
Sorry, I don't get this. 
Initially mDirty is false, if it was never specified with a flag.
Then IsDefaultValue() -> false, is that right?

Reply:
Arf, good catch, yes, forgot to flip the condition..
And as you might have guessed, I don't have a repro case for when this can happen :upside_down_face: